### ISSUE TITLE
Set default options for dispatchCustomEvent

### ISSUE THREAD

**Issue by fbring** [trivago/melody#126]
#### Explain the problem
The usage of `dispatchCustomEvent` accepts 3 arguments:
1. eventName 
2. details
3. options

I think that in almost every case `dispatchCustomEvent` will be used for communicating with a parent component and therefore we need to set the `bubbles: true` option manually.

I searched in our current codebase for usage of `dispatchCustomEvent`:
- 6/6 use `bubbles: true`
- 4/6 use `cancelable: true`

#### Expected Behaviour
I would like to see at least the `bubbles: true` as a default option. Maybe it makes sense to set `cancelable: true` as well.

#### Actual Behaviour
Set all options manually inside the options object.

**Comment by byara** [comment]
Isn't that a breaking change?

**Comment by pago** [comment]
I'm wondering whether we really should be overwriting the DOM here. Making `bubbles` explicit has some benefits for readability (you clearly communicate the intend). I'd have a hard time to identify any valid use case for a non-bubbling event. So you could also say its a bugfix.
Theoretically `bubbles: false` might be useful for reusable ref-handlers but the ergonomics of those are currently not good enough so I doubt anybody uses that.

=> To me it's a bugfix. `bubbles: false` is very unlikely to have the expected behaviour and is most likely a cause of bugs. @byara do you agree?

What I do know for sure is that we shouldn't make `cancelable` a default. I doubt that we have a single line of code dealing with an event that has been cancelled. And if you don't deal with that case then there's no sense in marking an event as cancelable.

**Comment by byara** [comment]
@pago agreed. I cannot come up with a case that I would set the `bubbles: false`.
